.github/workflows/release-prod.yml

```
@@ -0,0 +1,92 @@
+    timeout-minutes: 90
+    env:
+      VPS_HOST: ${{ secrets.VPS_HOST }}
+      VPS_USER: ${{ secrets.VPS_USER }}
+      VPS_SSH_KEY: ${{ secrets.VPS_SSH_KEY }}
+      PROD_HOST: ${{ secrets.PROD_HOST }}
+    steps:
+      - uses: actions/checkout@v4
+
+      # Backup pré-deploy (DB + uploads)
+      - name: Pre-deploy backup
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:47: Double quote to prevent globbing and word splitting

(shellcheck)